app/controllers/comment_controller.php

```diff
@@ -0,0 +1,64 @@
+<?php
+class CommentController extends AppController
+{
+    function __construct($name)
+    {
+        parent::__construct($name);
+
+        if (!is_logged_in()) 
+        {
+            redirectTo(url('user/login'));
+        }
+    }
+
+    //Display thread and it's comments
+    public function view()
```

Comment:
Edited. Thank you.

*Update*
As mentioned before, the constructor for this class was removed. Thank you.